### ISSUE TITLE
TypeError when running the wifi_monitor example

### ISSUE THREAD

**Issue by winfr34k** [seveas/python-networkmanager#36]
Hi!

I was trying to use the current version of your library for a project on a Debian 8.7 (NetworkManager: 0.9.10.0-7) host.

While running the wifi_monitor example, I encountered this:
```
ERROR:dbus.connection:Exception in handler for D-Bus signal:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 230, in maybe_handle_message
    self._handler(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/NetworkManager.py", line 77, in handle_signal
    receiver(sender, *(sargs + rargs), **rkwargs)
TypeError: can only concatenate list (not "tuple") to list
```

I have since figured out that 'rargs` seems to be a tuple at that point. 

What can I do to fix this?

**Comment by rusty71** [comment]
Confirmed; I 'fixed' this by changing 'rargs' to 'list(rargs)'
Not sure if this is a valid fix,

**Comment by seveas** [comment]
That's almost a correct fix :)

Try with commit aea55e8. I'd also appreciate it if you could test restarting networkmanager while that example is running. It should give you some errors during the restart but happily continue on afterward.

**Comment by winfr34k** [comment]
Hi!

Thanks. This fixed the problems for me.
As per your request, I've also tested the restart of NetworkManager whilst the example was running, but it doesn't continue working. The exception tree starts with: 

```ERROR:dbus.proxies:Introspect error on :1.6:/org/freedesktop/NetworkManager/Devices/3: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name :1.6 was not provided by any .service files```

And fails thereon after. The program still "works" in the sense that it doesn't fully crash, but I don't get any new events fired.

--
Best regards
Thomas

**Comment by seveas** [comment]
The restart handling is now fixed in b15dc84
